apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -117,6 +117,7 @@ export default function RubricContainer({
           studentUserId={studentLevelInfo && studentLevelInfo['user_id']}
           visible={selectedTab === TAB_NAMES.SETTINGS}
           refreshAiEvaluations={fetchAiEvaluations}
+          rubric={rubric}
```

Comment:
Since we're passing the whole rubric down, maybe we can do away with the rubricId prop?